### PR TITLE
test case for openclpow module

### PR DESCRIPTION
**Pull request by navjotcis**
written test case for testing openclpow module functionality.

### CODE REVIEW

**Review by PeterSurda — CHANGES_REQUESTED** [review]
Good start, fix CQ, add `@skipTestIf` so that it skips it if opencl isn't avaialble.

File: src/tests/test_openclpow.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+from struct import pack, unpack
+
+
+class TestOpenClPow(unittest.TestCase):
+    """
+    Main opencl test case
+    """
+
+    @unittest.skipIf(not openclpow.libAvailable,
```
Comment (PeterSurda) [inline src/tests/test_openclpow.py:17]: @unittest.skipIf(not openclpow.enabledGpus,
                                "No GPUs found / enabled")

**Review by PeterSurda — CHANGES_REQUESTED** [review]
change the skip decorator, otherwise ok

**Comment by PeterSurda** [comment]
Actually, now that I think about it, it's more complicated. You need to put `initCL` into `setupClass`.

**Comment by navjotcis** [comment]
in which module `setupClass` is present?

**Review by PeterSurda — APPROVED** [review]

**Comment by g1itch** [comment]
I don't understand why is this test case subclassed from TestProcessProto.

**Comment by PeterSurda** [comment]
@g1itch You're right. I'll request a fix.

**Comment by g1itch** [comment]
> @g1itch You're right. I'll request a fix.

ec6f078

**Pull request merged**